internal/wasm/host.go

```diff
@@ -156,6 +172,21 @@ func addFuncs(
 		if hf, ok := v.(*HostFunc); ok {
 			nameToFunc[hf.Name] = hf
 			funcNames = append(funcNames, hf.Name)
+		} else if pf, ok := v.(*ProxyFunc); ok {
+			proxiedIdx := len(funcNames)
+			if proxiedIdx > 255 {
```

Comment:
if the MSB is set, the leb128 unsigned doesn't match the literal byte
```suggestion
			if proxiedIdx > 127 {
```